### PR TITLE
[poppler] Disable clang-format check

### PR DESCRIPTION
**Pull request by NancyLi1013**
**Describe the pull request**

- What does your PR fix? Fixes #

`clang-format.exe` cannot be found now, So disable clang-format.
```
'clang-format' is not recognized as an internal or external command,
operable program or batch file.
'true' is not recognized as an internal or external command,
operable program or batch file.
```

### CODE REVIEW

**Review by JackBoosY — APPROVED** [review]

File: ports/poppler/portfile.cmake [hunk 1]
```diff
@@ -14,6 +14,11 @@ vcpkg_find_acquire_program(GPERF)
 get_filename_component(GPERF_PATH ${GPERF} DIRECTORY)
 vcpkg_add_to_path(${GPERF_PATH})
 
+vcpkg_find_acquire_program(CLANG_FORMAT)
+get_filename_component(CLANG_FORMAT_PATH ${CLANG} DIRECTORY)
```
Comment (NancyLi1013) [inline ports/poppler/portfile.cmake:18]:
```suggestion
get_filename_component(CLANG_FORMAT_PATH ${CLANG_FORMAT} DIRECTORY)
```

File: ports/poppler/portfile.cmake [hunk 2]
```diff
@@ -14,6 +14,11 @@ vcpkg_find_acquire_program(GPERF)
...
+vcpkg_add_to_path(${CLANG_FORMAT_PATH})
+
+
```
Comment (NancyLi1013) [inline ports/poppler/portfile.cmake:21]:
```suggestion
```

**Comment by NancyLi1013** [comment]
The regressions caused by gettext will be fixed by PR https://github.com/microsoft/vcpkg/pull/15740

**Comment by NancyLi1013** [comment]
cc @BillyONeal

**Comment by NancyLi1013** [comment]
Currently, I have added `clang-format`. Seems it needs to be installed on our CI machine.

```
CMake Error at scripts/cmake/vcpkg_find_acquire_program.cmake:566 (message):
  Could not find clang-format.  Please install it via your package manager:

      sudo apt-get install clang-format
```

**Comment by BillyONeal** [comment]
Rather than trying to provide clang-format I think we should be patching out whatever uses it in poppler's build system.

**Comment by NancyLi1013** [comment]
Thanks for your suggestion @BillyONeal. I will revert this changes about clang-format.

**Review by JackBoosY — APPROVED** [review]

**Pull request merged**